package.json

```diff
@@ -35,8 +35,7 @@
     "test": "npm run test:unit && npm run test:resources",
     "test:debug": "mocha \"src/**/*.test.ts\"",
     "test:resources": "mocha \"resources/**/*.test.ts\"",
-    "test:unit": "nyc mocha \"src/**/*.test.ts\" -- --reporter=xunit --reporter-options output=reports/generator.xml",
-    "version": "oclif-dev readme && git add README.md"
```

Comment:
This seems to just add the file to staging??

Reply:
The `version` script is a hook for the `npm version` command, which creates a commit.

Reply:
ah.. ok. I think we can still have it as a separate script not tied to version for convinience

Reply:
That's the existing `docs` command. 😉